exasol_script_languages_developer_sandbox/lib/ansible/ansible_repository.py

```diff
@@ -1,7 +1,9 @@
-import logging
 from pathlib import Path
 import importlib_resources as ir
 import exasol_script_languages_developer_sandbox.runtime.ansible
+from exasol_script_languages_developer_sandbox.lib.logging import get_status_logger, LogType
+
+LOG = get_status_logger(LogType.ANSIBLE)
```

Comment:
A common pattern for creating logger is to use the module name , is there a specific reason you are using specific categories

Reply:
The module name would be actually the file name, however this is already automatically printed by using the RichHandler.
I attached a screenshot for better understanding.
I create a logger per each submodule, which helps to communicate to the user which step the tool is currently doing.

Reply:
Also, I need to have a list of all custom loggers for which I want to set the log-level to (at least) info.